### PR TITLE
Feature/regex helpers

### PR DESCRIPTION
**Pull request by pmelkowski**

### CODE REVIEW

File: src/main/java/io/knotx/handlebars/helpers/regex/EachSplitHelper.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 Cognifide Limited
```
Comment (malaskowski) [inline src/main/java/io/knotx/handlebars/helpers/regex/EachSplitHelper.java]: Actually this should be:
> `Copyright (C) 2018 Knot.x Project`

It looks that it is invalid in other files in this project and should be fixed in another PR.

**Review by malaskowski — CHANGES_REQUESTED** [review]

File: src/main/java/io/knotx/handlebars/helpers/regex/EachSplitHelper.java (hunk 1 above)
Comment (pmelkowski) [reply src/main/java/io/knotx/handlebars/helpers/regex/EachSplitHelper.java]: Fixed for the two regex classes. Will create separate branch for the others.

Comment (malaskowski) [reply src/main/java/io/knotx/handlebars/helpers/regex/EachSplitHelper.java]: Thank you

**Review by malaskowski — APPROVED** [review]
